app/controllers/admin/project_steps_controller.rb

```diff
@@ -157,4 +163,10 @@ def render_modal_content(status = 200)
       context: params[:context]
     }
   end
+
+  def render_duplicate_modal_content(status = 200)
```

Comment:
This doesn't need to be a separate method since it's only used in one place and it's just one line. I would move it back to show_duplicate.

You can also remove the status param as it's not being used I think. The default status for `render` is 200.